packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -117,6 +117,7 @@ function PreviewWrapper( { item, onClick, ariaDescribedBy, children } ) {
 
 function Preview( { item, viewType } ) {
 	const descriptionId = useId();
+	const description = item.description || item?.excerpt?.raw;
```

Comment:
I know this was from before, but it surprized me to see `raw` and not `rendered` here..

Reply:
Same :) I think there's some consistency work to be done on the REST API.